radicle-crdt/src/clock.rs

```diff
@@ -0,0 +1,39 @@
+use serde::{Deserialize, Serialize};
+
+use crate::ord::Max;
+
+/// Lamport clock.
+#[derive(Debug, Default, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
+#[serde(transparent)]
+pub struct LClock {
```

Comment:
suggestion: call it `Lamport`. It can be imported as `Lamport` or `clock::Lamport`. Instead of guessing what the `L` means in `LClock` :)

Reply:
Good idea.